pages/linux/tshark.md

```diff
@@ -0,0 +1,31 @@
+# tshark
+
+> Packet analysis tool, cLI version of wireshark.
+
+- Monitor everything on localhost:
+
+`tshark`
+
+- Filter captured packets by method:
+
+`tshark -Y 'http.request.method == "GET"'`
+
+- Decode the current port with a specific protocol (ex: http):
+
+`tshark -d tcp.port==8888,http`
+
+- Format of captured output:
+
+`tshark -T json`
```

Comment:
```suggestion
`tshark -T {{json|text|ps|…}}`
```

Use `{{...}}` for values controlled by the user, also specifying more alternatives seems more useful.